clang/test/CodeGen/attr-target-clones-riscv-invaild.c

```diff
@@ -0,0 +1,8 @@
+// RUN: not %clang_cc1 -triple riscv64 -target-feature +i -emit-llvm -o - %s 2>&1 | FileCheck %s --check-prefix=CHECK-UNSUPPORT-OS
+
+// CHECK-UNSUPPORT-OS: error: target_clones is currently only supported on Linux
```

Comment:
Do we emit this error for target_version too?

Reply:
Added.